### PR TITLE
Added footer message for cf-component-card/CardContent

### PR DESCRIPTION
**Pull request by thellimist**
**This is the result**
![alt text][image1]

I added footer message because it was missing in cf-ui and was present in cloudflare.com

**Example of footer messages in cloudflare.com**
![alt text][image2]

[image1]: http://i.imgur.com/DpY9fzl.png 
[image2]: http://i.imgur.com/3o1fqa5.png

### CODE REVIEW

**Comment by jwineman** [comment]
@wyuenho @marksteyn @terinjokes @toekneestuck

**Comment by wyuenho** [comment]
LGTM

**Comment by jwineman** [comment]
What about cf-component-text for the the footer text?
https://cloudflare.github.io/cf-ui/#cf-component-text
```
<Text type="muted">Hello World</Text>
```

Do we care if our components have dependencies on each other?
Are there any examples in the app where we don't use this space for displaying a timestamp of the last time the card was modified?  If so is it worth renaming this to something more specific to its function?  Ex timestamp?

**Comment by wyuenho** [comment]
Calling it footer or foot-note is fine. I like using `<Text>` better.

screen shot 2016-08-23 at 18 21 14

File: packages/cf-util-text/src/text.js [hunk 1]
```diff
@@ -4,9 +4,9 @@ function getText(id) {
   if (!text[id]) {
     console.error('Missing text for: "' + id + '"');
     return id;
-  } else {
-    return text[id];
-  }
+  } 
+  
+  return text[id];
```
Comment (jwineman) [inline packages/cf-util-text/src/text.js]: Why did this change?

Comment (wyuenho) [inline packages/cf-util-text/src/text.js]: Linter probably

File: packages/cf-component-copyable-textarea/src/CopyableTextarea.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ class CopyableTextarea extends React.Component {
     let success;
     try {
       success = document.execCommand('copy');
-    } catch(e) {
+    } catch(exception) {
```
Comment (wyuenho) [inline packages/cf-component-copyable-textarea/src/CopyableTextarea.js]: `e` is the convention, there are no exceptions in JS, just errors.

**Comment by thellimist** [comment]
@wyuenho Edited

**Comment by jwineman** [comment]
Still not using `<Text>`

**Comment by wyuenho** [comment]
@thellimist say he couldn't get this package depend on cf-component-text

**Comment by jwineman** [comment]
Why not?

**Comment by thejameskyle** [comment]
Adding a cross-dependency: 

1. Add the `"name": "^version"` to `package.json`
2. Run `npm run bootstrap` again

**Comment by thejameskyle** [comment]
*vanishes into the night*

**Pull request merged**